scripts/REST_Axion_PlotResonances.C

```diff
@@ -0,0 +1,123 @@
+///////////////////////////////////////////////////////
+/// \file      AxionPlotResonances.C
+/// \brief     This script plots the transmission probability as a function of the axion mass for a given gas
+/// and axion energy until a maximum axion mass.
+// Arguments by default are (in order):
+// -ma_max = 0.1 eV (axion mass)
+// -Ea = 4.2 keV (axion energy)
+//  gasName = "He" (Gas name)
+//  vacuum = true (Vacuum added to the sum or not)
+//  n_ma = 100000 (Number of points to be plotted)
+//
+// Example of usage:
+// restRoot -l
+// .L AxionPlotResonances.C
+// AxionPlotResonances(0.1, 4.2, "He", true, 100000)
+//
+///////////////////////////////////////////////////
+/// \date       11/12/2023
+/// \author     Francisco R. Candon
+///////////////////////////////////////////////////
+
+int REST_Axion_PlotResonances(double ma_max = 0.1, double Ea = 4.2, std::string gasName = "He",
+                              Bool_t vacuum = true, int n_ma = 100000) {
+    TRestAxionField* ax = new TRestAxionField();
+
+    // m_a min to be considered
+    double ma_min = 0;  // eV
+
+    std::pair<std::vector<double>, std::vector<double>> pair =

```

Comment:
The return value needs to be adapted to `std::vector < std::pair < double, double> >`